nuget/Microsoft.Windows.CsWinRT.nuspec

```diff
@@ -36,12 +36,22 @@
     <file src="$winrt_host_x86$" target ="runtimes\win-x86\native"/>
     <file src="$winrt_host_arm$" target ="runtimes\win-arm\native"/>
     <file src="$winrt_host_arm64$" target ="runtimes\win-arm64\native"/>
-    <file src="$winrt_host_resource_x64$" target ="runtimes\win-x64\native\en-us"/>
-    <file src="$winrt_host_resource_x86$" target ="runtimes\win-x86\native\en-us"/>
-    <file src="$winrt_host_resource_arm$" target ="runtimes\win-arm\native\en-us"/>
-    <file src="$winrt_host_resource_arm64$" target ="runtimes\win-arm64\native\en-us"/>
     <file src="$winrt_shim$" target ="lib\net5.0\"/>
+
+    <!-- Localized resources for error strings --> 
     
+    <file src="..\src\Authoring\WinRT.Host\MUI\**" target ="runtimes\win-x64\native"/>
+    <file src="..\src\Authoring\WinRT.Host\MUI\**" target ="runtimes\win-x86\native"/>
+    <file src="..\src\Authoring\WinRT.Host\MUI\**" target ="runtimes\win-arm\native"/>
+    <file src="..\src\Authoring\WinRT.Host\MUI\**" target ="runtimes\win-arm64\native"/>
+    <file src="$winrt_host_resource_x64$" target ="runtimes\win-x64\native\en-US"/>
```

Comment:
What is the difference between this and the above ones again?

Reply:
Is this an optimization for en-US?

Reply:
The MUI folder in the repo just has folders like "fr-FR". So we're matching that pattern by putting the default ones (built when the project builds) in the english folder next to the french one.